packages/element-lib/package.json

```diff
@@ -38,6 +38,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@transmute/element-core": "^0.0.0",
```

Comment:
Is this used somewhere?

Reply:
another error, from testing, I am removing it.